templates/partials/financial-summary.html

```diff
@@ -1,7 +1,9 @@
   {% if related_committees %}
       <div class="page-section__intro">
         <h2 id="section-1-header" tabindex="0">Financial Summary</h2>
-        <p class="text--lead">Get the full picture of all of the money received and spent by this candidate’s principal and authorized committees.</p>
+        <p class="text--lead">This page provides an overview of all the money received and spent by the committees associated with this candidate.</p>
+        <p class="text--lead">Candidates are directly responsible for their <strong>principal campaign committees</strong> and <strong>authorized campaign committees</strong>. Candidates are associated with other types like <strong>leadership PACs</strong> and <strong>joint fundraising committees</strong>, but because those committees are not exclusively for the candidate's campaign activity, this page will not include their information.</p>
```

Comment:
Joint committees are on this page and I would love to have leadership pacs on this page in the future.